tap_marketo/__init__.py

```diff
@@ -58,6 +60,8 @@ def get_start(entity):
                       giveup=lambda e: e.response is not None and
                       400 <= e.response.status_code < 500,
                       factor=2)
+@backoff.on_exception(backoff.expo,
+                      RateLimitExceededException)
```

Comment:
I wonder if it might be better to improve the giveup function and include the RateLimitException in line 58 along with the RequestException. The giveup would need to be something like

```
lambda e: getattr(e, "response", None) is not None and ...
```

That way it would handle the fact that a RateLimitExceededException doesn't have a `response` attr.